.github/workflows/devbuild.yml

```diff
@@ -258,7 +272,20 @@ jobs:
             # GitHub Action macOS 13 has a problem with python3
             brew list -1 | grep python | while read formula; do brew unlink $formula; brew link --overwrite $formula; done;
           fi
-          brew install qt6
+
+          # brew install qt6
+          # # This has been replaced by the 'Install Qt' section below to manage qt6 versioning independently from the OS.
```

Comment:
Added explanation for disabling here.

Reply:
Please move the explanation above the code commented out.  Please avoid doubling the commenting mark like `# # ...`